.github/strategy/adapters.json

```diff
@@ -0,0 +1,67 @@
+{
+  "adapter": [
```

Comment:
We'll probably want to add writing to this file to the `make new` command.

Reply:
I agree, but we should probably use [`jq`](https://stedolan.github.io/jq/) instead of `sed` for transforming JSON.